PocketKit/Sources/PocketKit/MyList/ArchivedItemsList/ArchivedItemsListViewModel.swift

```diff
@@ -379,9 +379,16 @@ extension ArchivedItemsListViewModel {
                         self?.selectCell(with: .filterButton(.all))
                     }
                 )
-                presentedTagsFilter?.$filterTagDetails.sink { [weak self] details in
-                    guard let self = self, let predicate = details?.0, let name = details?.1 else { return }
-                    self.updateSnapshotForTagFilter(with: name, and: predicate)
+                presentedTagsFilter?.$selectedTag.sink { [weak self] selectedTag in
+                    guard let selectedTag = selectedTag else { return }
+                    let predicate: NSPredicate
+                    switch selectedTag {
+                    case .notTagged:
+                        predicate = NSPredicate(format: "tags.@count = 0")
+                    case .tag(let name):
+                        predicate = NSPredicate(format: "%@ IN tags.name", name)
+                    }
```

Comment:
thought about adding this to Archive Service instead, but maybe should be consistent with SavedItems